modules/parquet/src/parquetjs/parser/decoders.ts

```diff
@@ -63,7 +63,11 @@ export async function decodeDataPages(
       continue;
     }
 
-    if (dictionary.length) {
+    const valueEncoding = getThriftEnum(
+      Encoding,
+      page.pageHeader.data_page_header?.encoding!
+    ) as ParquetCodec;
+    if (dictionary.length && valueEncoding !== 'PLAIN') {
```

Comment:
Are there other type of encodings so that it makes sense to be more specific? 
```suggestion
    if (dictionary.length && valueEncoding === 'RLE_DICTIONARY') {
```

Maybe add the comment in the PR above this if statement?

Reply:
Done